configure

```diff
@@ -2050,6 +2050,23 @@ fi
 print_config "strndup" "$strndup"
 
 ##########################################
+# <valgrind/valgrind.h> probe
+if test "$valgrind_valgrind_h" != "yes" ; then
+  valgrind_valgrind_h="no"
+fi
+cat > $TMPC << EOF
+#include <valgrind/valgrind.h>
+int main(int argc, char **argv)
+{
+  return 0;
+}
+EOF
+if compile_prog "" "" "valgrind_valgrind_h"; then
+  valgrind_valgrind_h="yes"
+  output_sym "HAVE_VALGRIND_VALGRIND_H"
```

Comment:
Don't we normally save the result into a variable and then towards the end of the makefile output_sym a CONFIG_* variables (e.g. CONFIG_VALGRIND) that we then act on later?

Reply:
Additionally, it's named differently than the others. We should make it CONFIG_VALGRIND_DEV or something.

Reply:
I had missed this comment. It should now have been addressed.